operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -171,14 +171,14 @@ spec:
 # **Jaeger-specific settings:
 # jaeger_service: The Kubernetes service name for jaeger. Kiali uses this to connect within the cluster to Jaeger.
 # service: The Kubernetes service name for Tracing. Kiali uses this to connect within the cluster to Tracing.
-# service_namespace: The Kubernetes namespace that holds the Jaeger/Tracing service (if empty, assumes the same value as deployment.namespace)
+# namespace: The Kubernetes namespace that holds the Jaeger/Tracing service (if empty, assumes the same value as deployment.namespace)
```

Comment:
please alphabetize - all settings are alphanumerically sorted .... as per my previous comment:

> (don't forget to put them in the proper sorted order -- now that its "namespace" and not "service_namespace" it will reorder what you have currently)